### PR TITLE
[ i18n sprint ] Moved translations from email templates to property files

### PR DESCRIPTION
**Pull request by litvinovg**
[Companion Vitro PR](https://github.com/vivo-project/Vitro/pull/334)
Extracted translations from email templates into property files

# Interested parties
@VIVO-project/vivo-committers

### CODE REVIEW

File: en_US/webapp/src/main/webapp/i18n/all_en_US.properties [hunk 1]
```diff
@@ -981,3 +981,31 @@ file_upload_predicate = predicate
...
+password_reset_pending_email_subject = ${siteName} reset password request
+
+password_reset_complete_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Password successfully changed.</strong>\n </p>\n\n <p>\n Your new password associated with ${userAccount.emailAddress} has been changed.\n </p>\n\n <p>\n Thank you.\n </p>
+password_reset_complete_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nPassword successfully changed.\n\nYour new password associated with ${userAccount.emailAddress}\nhas been changed
+
+confirm_email_changed_email_html_text = <p>\n Hi, ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n You recently changed the email address associated with\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Thank you.\n </p>
```
Comment (chenejac) [inline en_US/webapp/src/main/webapp/i18n/all_en_US.properties]: I would move comma (,) after the last name:
Hi Dragan Ivanovic,

Actually, I think we should use here Dear , not Hi (as line 991). If we don't want to risk with language practices for using comma, we might also adopt as in line 991

Dear Dragan Ivanovic:

**Review by chenejac — CHANGES_REQUESTED** [review]
It works for me. Please check one my comment for usage of comma after Hi

File: en_US/webapp/src/main/webapp/i18n/all_en_US.properties (hunk 1 above)
Comment (litvinovg) [reply en_US/webapp/src/main/webapp/i18n/all_en_US.properties]: Fixed

**Review by chenejac — CHANGES_REQUESTED** [review]
this should be fixed for all languages I think. For sure, it should be fixed for Serbian language

File: sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties [hunk 2]
```diff
@@ -972,3 +972,31 @@ label.dateTimeWithPrecision.day_capitalized = Dan
...
+password_reset_pending_email_subject = ${siteName} zahtev za reset lozinke
+
+password_reset_complete_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Lozinka je uspešno promenjena.</strong>\n </p>\n\n <p>\n Vaša lozinka povezana sa nalogom ${userAccount.emailAddress} je promenjena.\n </p>\n\n <p>\n Hvala Vam.\n </p>\n
+password_reset_complete_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nLozinka je uspešno promenjena.\n\nVaša lozinka povezana sa nalogom ${userAccount.emailAddress} je promenjena.\n\nHvala Vam.
+
+confirm_email_changed_email_html_text = <p>\n Pozdrav, ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Nedavno ste promenili email adresu povezanu sa nalogom za korisnika\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Hvala Vam.\n </p>
```
Comment (chenejac) [inline sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties]:
```suggestion
confirm_email_changed_email_html_text = <p>\n Poštovani ${userAccount.firstName} ${userAccount.lastName}:\n </p>\n\n <p>\n Nedavno ste promenili email adresu povezanu sa nalogom za korisnika\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Hvala Vam.\n </p>
```

File: sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties [hunk 3]
```diff
@@ -972,3 +972,31 @@ label.dateTimeWithPrecision.day_capitalized = Dan
...
+confirm_email_changed_email_plain_text = Pozdrav, ${userAccount.firstName} ${userAccount.lastName}\n\nNedavno ste promenili email adresu povezanu sa nalogom za korisnika\n${userAccount.firstName} ${userAccount.lastName}\n\nHvala Vam.
```
Comment (chenejac) [inline sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties]:
```suggestion
confirm_email_changed_email_plain_text = Poštovani ${userAccount.firstName} ${userAccount.lastName}:\n\nNedavno ste promenili email adresu povezanu sa nalogom za korisnika\n${userAccount.firstName} ${userAccount.lastName}\n\nHvala Vam.
```

**Review by chenejac — CHANGES_REQUESTED** [review]

File: sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties [hunk 4]
```diff
@@ -972,3 +972,31 @@ label.dateTimeWithPrecision.day_capitalized = Dan
...
+
+application_error_email_html_text = <html>\n <head>\n <title>${subject!}</title>\n </head>\n <body>\n <p>\n ${i18n().error_occurred_at(siteName!,datetime!)}\n </p>\n\n <p>\n <strong>${i18n().requested_url}:</strong> ${requestedUrl!}\n </p>\n\n <p>\n <#if errorMessage?has_content>\n <strong>${i18n().error_message}:</strong> ${errorMessage!}\n </#if>\n </p>\n\n <p>\n <strong>${i18n().stack_trace}</strong> (${i18n().trace_available(siteName!)}):\n <pre>${stackTrace!}</pre>\n </p>\n\n <#if cause?has_content>\n <p><strong>${i18n().caused_by}:</strong>\n <pre>${cause!}</pre>\n </p>\n </#if>\n\n </body>\n</html>
+
+application_error_email_plain_text = ${i18n().error_occurred_at(siteName!,datetime!)}\n\n${i18n().requested_url}: ${requestedUrl!}\n\n<#if errorMessage?has_content>\n ${i18n().error_message}: ${errorMessage!}\n</#if>\n\n${i18n().stack_trace} (${i18n().trace_available(siteName!)}):\n${stackTrace!}\n\n<#if cause?has_content>\n${i18n().caused_by}:\n${cause!}\n</#if>
+
+application_error_email_subject = ${i18n().error_occurred(siteName!)}
```
Comment (chenejac) [inline sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties]: @litvinovg can you please try to improve those two lines in accordance with our discussion: to copy here text from error_occurred translation, and to move this i18n method invocation out from the property file

**Review by chenejac — CHANGES_REQUESTED** [review]

File: sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties (hunk 4 above)
Comment (litvinovg) [reply sr_Latn_RS/webapp/src/main/webapp/i18n/all_sr_Latn_RS.properties]: Done

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 5]
```diff
@@ -951,3 +951,31 @@ label.dateTimeWithPrecision.day_capitalized = Tag
...
+password_created_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nPasswort erfolgreich erstellt.\n\nIhr Passwort für den Account ${userAccount.emailAddress}\nwurde erstellt.\n\nVielen Dank.
+
+application_error_email_html_text = <p>Zum Zeitpunk ${datetime!} ist ein Fehler auf der Webseite ${siteName!} aufgetreten.</p>\n\n <p>\n <strong>${i18n().requested_url}:</strong> ${requestedUrl!}\n </p>\n\n <p>\n <#if errorMessage?has_content>\n <strong>${i18n().error_message}:</strong> ${errorMessage!}\n </#if>\n </p>\n\n <p>\n <strong>${i18n().stack_trace}</strong> (${i18n().trace_available(siteName!)}):\n <pre>${stackTrace!}</pre>\n </p>\n\n <#if cause?has_content>\n <p><strong>${i18n().caused_by}:</strong>\n <pre>${cause!}</pre>\n </p>\n </#if>
+
+application_error_email_plain_text = Zum Zeitpunk ${datetime!} ist ein Fehler auf der Webseite ${siteName!} aufgetreten.\n${i18n().requested_url}: ${requestedUrl!}\n\n<#if errorMessage?has_content>\n ${i18n().error_message}: ${errorMessage!}\n</#if>\n\n${i18n().stack_trace} (${i18n().trace_available(siteName!)}):\n${stackTrace!}\n\n<#if cause?has_content>\n${i18n().caused_by}:\n${cause!}\n</#if>
```
Comment (chenejac) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]: here are still i18n().requested_url, etc.

**Review by chenejac — CHANGES_REQUESTED** [review]

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties (hunk 5 above)
Comment (litvinovg) [reply de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]: Fixed that

File: fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties [hunk 6]
```diff
@@ -965,3 +965,31 @@ label.dateTimeWithPrecision.day_capitalized = Jour
 label.dateTimeWithPrecision.hour_capitalized = Heure
 label.dateTimeWithPrecision.minutes_capitalized = Minutes
 label.dateTimeWithPrecision.seconds_capitalized = Secondes
+
```
Comment (michel-heon) [inline fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties:968]:
```suggestion


acct_created_external_only_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nFélicitations!\n\nNous avons créé votre nouveau compte VIVO associé à\n${userAccount.emailAddress}.\n\nMerci!\n
acct_created_external_only_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Félicitations!</strong>\n </p>\n\n <p>\n Nous avons créé votre nouveau compte VIVO associé à ${userAccount.emailAddress}.\n </p>\n\n <p>\n Merci!\n </p>

acct_created_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nFélicitations!\n\nNous avons créé votre nouveau compte sur le site ${siteName},\nassociés à ${userAccount.emailAddress}.\n\nSi vous n'avez pas demandé ce nouveau compte, vous pouvez ignorer ce courriel..\nCette demande expirera si aucune action n'est entreprise avant 30 jours.\n\nCollez le lien ci-dessous dans la barre d'adresse de votre navigateur pour créer votre mot de passe\npour votre nouveau compte en utilisant notre serveur sécurisé.\n\n${passwordLink}\n\nMerci!
acct_created_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Félicitations!</strong>\n </p>\n\n <p>\n Nous avons créé votre nouveau compte sur le site ${siteName}, associés à ${userAccount.emailAddress}.\n </p>\n\n <p>\n Si vous n'avez pas demandé ce nouveau compte, vous pouvez ignorer ce courriel.\nCette demande expirera si aucune action n'est entreprise dans un délai de 30 jours.\n </p>\n\n <p>\n Cliquez sur le lien ci-dessous pour créer votre mot de passe pour votre nouveau compte en utilisant notre serveur sécurisé.\n </p>\n\n <p>\n <a href="${passwordLink}" title="mot de passe">${passwordLink}</a>\n </p>\n\n <p>\n Si le lien ci-dessus ne fonctionne pas, vous pouvez copier et coller le lien directement dans la barre d'adresse de votre navigateur..\n </p>\n\n <p>\n Merci!\n </p>

password_reset_pending_email_html_text = <p>\n Cher ${userAccount.firstName} ${userAccount.lastName}:\n </p>\n\n <p>\n Nous avons reçu une demande de réinitialisation du mot de passe de votre site web. ${siteName} compte (${userAccount.emailAddress}).\n </p>\n\n <p>\n Veuillez suivre les instructions ci-dessous pour procéder à la réinitialisation de votre mot de passe..\n </p>\n\n <p>\n Si vous n'avez pas demandé ce nouveau compte, vous pouvez ignorer ce courrier.\nCette demande expirera si elle n'est pas suivie d'effet dans un délai de 30 jours.\n </p>\n\n <p>\n Cliquez sur le lien ci-dessous ou collez-le dans la barre d'adresse de votre navigateur pour réinitialiser votre mot de passe.\n en utilisant notre serveur sécurisé.\n </p>\n\n <p><a href="${passwordLink}" title="mot de passe">${passwordLink}</a> </p>\n\n <p>Merci.</p>
password_reset_pending_email_plain_text = Cher ${userAccount.firstName} ${userAccount.lastName}:\n\nNous avons reçu une demande de réinitialisation du mot de passe de votre site web. ${siteName} compte\n(${userAccount.emailAddress}).\n\nVeuillez suivre les instructions ci-dessous pour procéder à la réinitialisation de votre mot de passe.\n\nSi vous n'avez pas demandé ce nouveau compte, vous pouvez ignorer ce courriel.\nCette demande expirera si elle n'est pas traitée dans un délai de 30 jours.\n\nCollez le lien ci-dessous dans la barre d'adresse de votre navigateur pour réinitialiser votre mot de passe.\nen utilisant notre serveur sécurisé.\n\n${passwordLink}\n\nMerci.\n
password_reset_pending_email_subject = ${siteName} demande de réinitialisation du mot de passe

password_reset_complete_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Le mot de passe a été changé avec succès.</strong>\n </p>\n\n <p>\n Votre nouveau mot de passe associé à ${userAccount.emailAddress} a été modifié.\n </p>\n\n <p>\n Merci.\n </p>
password_reset_complete_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nLe mot de passe a été changé avec succès.\n\nVotre nouveau mot de passe associé à ${userAccount.emailAddress}\na été modifié

confirm_email_changed_email_html_text = <p>\n Bonjour, ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Vous avez récemment changé l'adresse courriel associée à\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n Merci.\n </p>
confirm_email_changed_email_plain_text = Bonjour, ${userAccount.firstName} ${userAccount.lastName}\n\nVous avez récemment changé l'adresse courriel associée à\n${userAccount.firstName} ${userAccount.lastName}\n\nMerci.

first_time_external_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Félicitations!</strong>\n </p>\n\n <p>\n Nous avons créé votre nouveau compte VIVO associé à ${userAccount.emailAddress}.\n </p>\n\n <p>\n Merci.\n </p>
first_time_external_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nCongratulations!\n\nNous avons créé votre nouveau compte VIVO associé à\n${userAccount.emailAddress}.\n\nMerci!

password_created_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Le mot de passe a été créé avec succès.</strong>\n </p>\n\n <p>\n Votre nouveau mot de passe associé à ${userAccount.emailAddress} a été créé.\n </p>\n\n <p>\n Merci.\n </p>
password_created_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nLe mot de passe a été créé avec succès.\n\nVotre nouveau mot de passe associé à ${userAccount.emailAddress}\na été créé.\n\nMerci..

application_error_email_html_text = <p>Une erreur s'est produite sur votre site ${siteName!} à ${datetime!}</p>\n\n <p>\n <strong>URL demandée:</strong> ${requestedUrl!}\n </p>\n\n <p>\n <#if errorMessage?has_content>\n <strong>Message d'erreur:</strong> ${errorMessage!}\n </#if>\n </p>\n\n <p>\n <strong>Trace de la pile</strong> (trace complète disponible dans le log):\n <pre>${stackTrace!}</pre>\n </p>\n\n <#if cause?has_content>\n <p><strong>Causé par:</strong>\n <pre>${cause!}</pre>\n </p>\n </#if>

application_error_email_plain_text = Une erreur s'est produite sur votre site ${siteName!} à ${datetime!}\n\nURL demandée: ${requestedUrl!}\n\n<#if errorMessage?has_content>\n Message d'erreur: ${errorMessage!}\n</#if>\n\nTrace de la pile (trace complète disponible dans le log):\n${stackTrace!}\n\n<#if cause?has_content>\nCausé par:\n${cause!}\n</#if>

application_error_email_subject = Une erreur s'est produite sur votre site ${siteName!}
```

**Review by chenejac — APPROVED** [review]

**Review by brianjlowe — APPROVED** [review]

File: de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 7]
```diff
@@ -951,3 +951,31 @@ label.dateTimeWithPrecision.day_capitalized = Tag
...
+
+password_reset_pending_email_html_text = <p>\n Sehr geehrte/r ${userAccount.firstName} ${userAccount.lastName}:\n </p>\n\n <p>\n Wir haben eine Anfrage erhalten, Ihr Passwort für Ihren Account (${userAccount.emailAddress}) auf ${siteName} zurückzusetzen.\n </p>\n\n <p>\n Bitte folgen Sie den Anweisungen unten, um mit dem Zurücksetzen Ihres Passworts fortzufahren.\n </p>\n\n <p>\n If you did not request this new account you can safely ignore this email.\n This request will expire if not acted upon within 30 days.\n </p>\n\n <p>\n\n Fügen Sie den untenstehenden Link in die Adressleiste Ihres Browsers ein,\n um Ihr Passwort über unseren Server zurückzusetzen.\n </p>\n\n <p><a href="${passwordLink}" title="password">${passwordLink}</a> </p>\n\n <p>Vielen Dank!</p>
```
Comment (bkampe) [inline de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties]:
```suggestion
password_reset_pending_email_html_text = <p>\n Sehr geehrte/r ${userAccount.firstName} ${userAccount.lastName}:\n </p>\n\n <p>\n Wir haben eine Anfrage erhalten, Ihr Passwort für Ihren Account (${userAccount.emailAddress}) auf ${siteName} zurückzusetzen.\n </p>\n\n <p>\n Bitte folgen Sie den Anweisungen unten, um mit dem Zurücksetzen Ihres Passworts fortzufahren.\n </p>\n\n <p>\n Wenn Sie diesen Account nicht beantragt haben, können Sie diese E-Mail ignorieren.\nDiese Anfrage erlischt automatisch nach 30 Tagen.\n </p>\n\n <p>\n\n Fügen Sie den untenstehenden Link in die Adressleiste Ihres Browsers ein,\n um Ihr Passwort über unseren Server zurückzusetzen.\n </p>\n\n <p><a href="${passwordLink}" title="password">${passwordLink}</a> </p>\n\n <p>Vielen Dank!</p>
```

**Review by bkampe — CHANGES_REQUESTED** [review]
One sentence was not translated to german

File: fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties [hunk 8]
```diff
@@ -965,3 +965,59 @@ label.dateTimeWithPrecision.day_capitalized = Jour
...
+password_created_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nPassword successfully created.\n\nYour new password associated with ${userAccount.emailAddress}\nhas been created.\n\nThank you.
+
+application_error_email_html_text = <p>Une erreur s'est produite sur votre site ${siteName!} à ${datetime!}</p>\n\n <p>\n <strong>URL demandée:</strong> ${requestedUrl!}\n </p>\n\n <p>\n <#if errorMessage?has_content>\n <strong>Message d'erreur:</strong> ${errorMessage!}\n </#if>\n </p>\n\n <p>\n <strong>Trace de la pile</strong> (trace complète disponible dans le log):\n <pre>${stackTrace!}</pre>\n </p>\n\n <#if cause?has_content>\n <p><strong>Causé par:</strong>\n <pre>${cause!}</pre>\n </p>\n </#if>
+
+application_error_email_plain_text = Une erreur s'est produite sur votre site ${siteName!} à ${datetime!}\n\nURL demandée: ${requestedUrl!}\n\n<#if errorMessage?has_content>\n Message d'erreur: ${errorMessage!}\n</#if>\n\nTrace de la pile (trace complète disponible dans le log):\n${stackTrace!}\n\n<#if cause?has_content>\nCausé par:\n${cause!}\n</#if>
+
+application_error_email_subject = Une erreur s'est produite sur votre site ${siteName!}
```
Comment (chenejac) [inline fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties]:
```suggestion

```

Comment (chenejac) [reply fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties]: In the French file we have duplicated label keys (firstly with labels translated in French, and then with labels translated in English). The last one are used in the UI (which is wrong).

**Review by chenejac — CHANGES_REQUESTED** [review]

**Review by bkampe — APPROVED** [review]
German translation looks fine

File: fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties [hunk 9]
```diff
@@ -965,3 +965,35 @@ label.dateTimeWithPrecision.day_capitalized = Jour
...
+acct_created_external_only_email_plain_text = ${userAccount.firstName} ${userAccount.lastName}\n\nCongratulations!\n\nWe have created your new VIVO account associated with\n${userAccount.emailAddress}.\n\nThanks!\n
+acct_created_external_only_email_html_text = <p>\n ${userAccount.firstName} ${userAccount.lastName}\n </p>\n\n <p>\n <strong>Congratulations!</strong>\n </p>\n\n <p>\n We have created your new VIVO account associated with ${userAccount.emailAddress}.\n </p>\n\n <p>\n Thanks!\n </p>
```
Comment (chenejac) [inline fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties]:
```suggestion

```

File: fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties [hunk 10]
```diff
@@ -965,3 +965,35 @@ label.dateTimeWithPrecision.day_capitalized = Jour
...
+password_reset_pending_email_plain_text = Cher ${userAccount.firstName} ${userAccount.lastName}:\n\nNous avons reçu une demande de réinitialisation du mot de passe de votre site web. ${siteName} compte\n(${userAccount.emailAddress}).\n\nVeuillez suivre les instructions ci-dessous pour procéder à la réinitialisation de votre mot de passe.\n\nSi vous n'avez pas demandé ce nouveau compte, vous pouvez ignorer ce courriel.\nCette demande expirera si elle n'est pas traitée dans un délai de 30 jours.\n\nCollez le lien ci-dessous dans la barre d'adresse de votre navigateur pour réinitialiser votre mot de passe.\nen utilisant notre serveur sécurisé.\n\n${passwordLink}\n\nMerci.\n
```
Comment (chenejac) [inline fr_CA/webapp/src/main/webapp/i18n/all_fr_CA.properties]:
```suggestion
password_reset_pending_email_html_text = <p>\n Cher ${userAccount.firstName} ${userAccount.lastName}:\n </p>\n\n <p>\n Nous avons reçu une demande de réinitialisation du mot de passe de votre site web pour le compte ${siteName} (${userAccount.emailAddress}).\n </p>\n\n <p>\n Veuillez suivre les instructions ci-dessous pour procéder à la réinitialisation de votre mot de passe..\n </p>\n\n <p>\n Si vous n'avez pas demandé ce nouveau compte, vous pouvez ignorer ce courrier.\nCette demande expirera si elle n'est pas suivie d'effet dans un délai de 30 jours.\n </p>\n\n <p>\n Cliquez sur le lien ci-dessous ou collez-le dans la barre d'adresse de votre navigateur pour réinitialiser votre mot de passe,\n en utilisant notre serveur sécurisé.\n </p>\n\n <p><a href="${passwordLink}" title="mot de passe">${passwordLink}</a> </p>\n\n <p>Merci.</p>
password_reset_pending_email_plain_text = Cher ${userAccount.firstName} ${userAccount.lastName}:\n\nNous avons reçu une demande de réinitialisation du mot de passe de votre site web pour le compte ${siteName} \n(${userAccount.emailAddress}).\n\nVeuillez suivre les instructions ci-dessous pour procéder à la réinitialisation de votre mot de passe.\n\nSi vous n'avez pas demandé ce nouveau compte, vous pouvez ignorer ce courriel.\nCette demande expirera si elle n'est pas traitée dans un délai de 30 jours.\n\nCollez le lien ci-dessous dans la barre d'adresse de votre navigateur pour réinitialiser votre mot de passe,\nen utilisant notre serveur sécurisé.\n\n${passwordLink}\n\nMerci.\n

```

**Review by chenejac — CHANGES_REQUESTED** [review]
@litvinovg I have added a couple of suggestions for improvement of French provided by Michel Heon.

**Review by chenejac — APPROVED** [review]

**Pull request merged**